src/recipes/ai-context/index.ts

```diff
@@ -56,11 +62,69 @@ const promptForPath = async (): Promise<string> => {
   return promptForPath()
 }
 
-export const run = async ({ args, command }: RunRecipeOptions) => {
+type IDE = {
+  name: string
+  command: string
+  rulesPath: string
+}
+const IDE: IDE[] = [
+  {
+    name: 'Windsurf',
+    command: 'windsurf',
+    rulesPath: IDE_RULES_PATH_MAP.windsurf,
+  },
+  {
+    name: 'Cursor',
+    command: 'cursor',
+    rulesPath: IDE_RULES_PATH_MAP.cursor,
+  },
+]
+
+const getPathByDetectingIDE = async (): Promise<string | null> => {
+  const getIDEFromCommand = (command: string): IDE | null => {
+    const match = IDE.find((ide) => command.includes(ide.command))
+    return match ?? null
+  }
+
+  async function getCommandAndParentPID(pid: number): Promise<{
```

Comment:
nit: Any reason for this function to be declared inside another function? Also we have function declarations and function expressions in the same scope, might be good to standardise.